content/master/operations/operation.md

```diff
@@ -0,0 +1,580 @@
+---
+title: Operation
+weight: 110
+state: alpha
+alphaVersion: 2.0
+description: Operations run function pipelines once to completion for operational tasks
+---
+
+An `Operation` runs a function pipeline once to completion to perform operational
+tasks that don't fit the typical resource creation pattern. Unlike compositions
+that continuously reconcile desired state, Operations focus on tasks like
+backups, rolling upgrades, configuration validation, and scheduled maintenance.
+
+## How operations work
+
+Operations are like Kubernetes Jobs - they run once to completion rather than
+continuously reconciling. Like compositions, Operations use function pipelines
+to implement their logic, but they're designed for operational workflows
+instead of resource composition.
+
+```yaml
+apiVersion: ops.crossplane.io/v1alpha1
+kind: Operation
+metadata:
+  name: backup-database
+spec:
+  mode: Pipeline
+  pipeline:
+  - step: create-backup
+    functionRef:
+      name: function-database-backup
+    input:
+      apiVersion: fn.crossplane.io/v1beta1
+      kind: DatabaseBackupInput
+      database: production-db
+      retentionDays: 30
+```
+
+When you create this Operation, Crossplane:
+
+1. **Validates** the operation and its function dependencies
+2. **Executes** the function pipeline step by step
+3. **Applies** any resources the functions create or change
+4. **Updates** the Operation status with results and completion state
+
+{{<hint "important">}}
+Operations are an alpha feature. You must enable them by adding
+`--enable-operations` to Crossplane's arguments.
+{{</hint>}}
+
+## Key characteristics
+
+- **Runs once to completion** (like Kubernetes Jobs)
+- **Uses function pipelines** (like Compositions)
+- **Can create or change any Kubernetes resources**
+- **Provides detailed status and output from each step**
+- **Supports retry on failure with configurable limits**
+
+## Operation functions vs composition functions
+
+Operations and compositions both use function pipelines, but with important
+differences:
+
+**Composition Functions:**
+- **Purpose**: Create and maintain resources
+- **Lifecycle**: Continuous reconciliation
+- **Input**: Observed composite resources
+- **Output**: Desired composed resources  
+- **Ownership**: Creates owner references
+
+**Operation Functions:**
+- **Purpose**: Perform operational tasks
+- **Lifecycle**: Run once to completion
+- **Input**: Required resources only
+- **Output**: Any Kubernetes resources
+- **Ownership**: Force applies without owners
```

Comment:
feels like this would be better surfaced as a table so you can see side by side comparison more easily

Reply:
odd, we do have tables in some places at least, e.g. https://docs.crossplane.io/v1.20/software/install/#feature-flags - maybe we can follow up later to see if we can get it working 🤓